service/models.py

```diff
@@ -164,28 +186,12 @@ def find_by_name(cls, name):
         return cls.query.filter(cls.name == name)
 
 
-"""
-Model
-------
-Product - A Prodcut available through the e-commerce service
-
-Attributes:
------------
-id (int) - the id of the prodcut 
-name (string) - the name of the product 
-price () - the price of the product 
-units (int) - the number of units available of a specific product 
-
-"""
-
 
 class Product(db.Model):
```

Comment:
I think we do not need to maintain a table of `Product`. Instead, we just need a `product_id`.